fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -1367,18 +1356,10 @@ void RocksDBMetrics::logStats(rocksdb::DB* db) {
 		ASSERT(db->GetIntProperty(property, &stat));
 		e.detail(name, stat);
 	}
-	/*
-	stat = readIterPool->numReadIteratorsCreated();
-	e.detail("NumReadIteratorsCreated", stat - readIteratorPoolStats["NumReadIteratorsCreated"]);
-	readIteratorPoolStats["NumReadIteratorsCreated"] = stat;
-	stat = readIterPool->numTimesReadIteratorsReused();
-	e.detail("NumTimesReadIteratorsReused", stat - readIteratorPoolStats["NumTimesReadIteratorsReused"]);
-	readIteratorPoolStats["NumTimesReadIteratorsReused"] = stat;
-	*/
 }
 
-void RocksDBMetrics::logMemUsagePerShard(std::string shardName, rocksdb::DB* db) {
-	TraceEvent e("ShardedRocksDBShardMemMetrics");
+void RocksDBMetrics::logMemUsage(rocksdb::DB* db) {
+	TraceEvent e("ShardedRocksDBMemMetrics");
```

Comment:
nit: add `SevDebug`? 

I am aware this is newly added, but consider:
Either pass in the debugID, or add a `detail("RocksDBPath", db->GetName())`

Reply:
I'd like to keep it as SevInfo, so we could get it in tests by default.

Reply:
The default severity is SevDebug for simulation test.